### ISSUE TITLE
Pip install via pypi

### ISSUE THREAD

**Issue by Shimwell** [openmc-dev/plotter#52]
Hi all,

Just looking around the repo and noticed it is 100% python.

Therefore I think would be fairly straight forward to make a conda install and pip install on PyPi for this tool 

What do you think, I am happy to help with this if the feature if it is of interest

**Comment by pshriwise** [comment]
Yeah I think that'd be great if you're up for it!

**Comment by paulromano** [comment]
Agreed, that would be great. It would be even better if we had OpenMC itself pip-installable so that we could properly state any version dependency, but that remains a further off task.

**Comment by Shimwell** [comment]
I think PR #54 helps automate this. Just need to remember to bump the version number when merging into the master branch

**Comment by Shimwell** [comment]
Had a quick go at the Conda Forge install as well but this would have to wait till the PyPi exists as the PyPi package SHA256 is needed. Alternatively GitHub releases can be used. I have put the Conda meta.yml here for now but we can move it to somewhere more openmc based https://github.com/Shimwell/staged-recipes/tree/add-openmc-plotter/recipes/openmc-plotter . Perhaps the openmc org repo could fork the staged-recipes as described in the contribution section https://conda-forge.org/#add_recipe

**Comment by kingyue737** [comment]
I've tried to install plotter by pip in `openmc-env` conda env which is created following openmc docs. However, several dependencies errors occur including what I've met one year ago https://openmc.discourse.group/t/qpenmc-plotter-import-error/774. 

Does it mean that currently openmc must be installed from source to use plotter?

**Comment by paulromano** [comment]
@kingyue737 I believe it should still work if you install OpenMC from conda. Can you be more specific about what dependency errors you're seeing?

**Comment by shimwell** [comment]
Just to note, the conda install is currently version 0.12.2 and this is a couple of months behind the GitHub develop branch.

**Comment by paulromano** [comment]
@shimwell That is true, but the version on PyPI should still work.

**Comment by AI-Pranto** [comment]
@paulromano If you use conda to install `openmc` and pip for `openmc-plotter`, you'll get the same error that @kingyue737 got a year ago because of version mismatch between pip (comes with pyside2-5.15) and conda (with pyside2-5.13.2).

**Comment by shimwell** [comment]
Perhaps we can pin the version of pyside to 2-5.12 in the setup.py https://github.com/openmc-dev/plotter/blob/180a3e929efda64a2ee148e38ff2b6c19ef71782/setup.py#L40

**Comment by AI-Pranto** [comment]
@shimwell still it's not working in conda :(
```bash
TypeError: 'Shiboken.ObjectType' object is not iterable
```
We need at least pyside-5.14 to solve this issue. https://bugreports.qt.io/browse/PYSIDE-1140

**Comment by shimwell** [comment]
I wonder if this might be worth another try due to the recent release of openmc 0.13.0 on conda.

This pip install worked for me in docker, assuming you have docker then this is the command I used to make a new image and run the enviroment

```bash
docker run -it continuumio/miniconda3
```
once inside docker these commands worked
```bash
conda install -c conda-forge openmc=0.13.0 -y
pip install openmc-plotter
```

**Comment by paulromano** [comment]
Closing this since `pip install` does work for the plotter. I've just created a separate issue (#92) regarding ability to install using `conda` directly.
